README.md

```diff
@@ -0,0 +1,33 @@
+# ROS AMR interoperability packages
+
+This repository is intended to host a collection of ROS packages to ease
+the integration of ROS based robots with different interoperability
+standards, with a focus on AMRs (Autonomous Mobile Robots).
+
+## Packages
+
```

Comment:
I prefer to avoid having two consecutive headings without text between them. Here you could introduce the section and whatever is general about the packages.